### PR TITLE
Add kg pm

### PR DESCRIPTION
**Pull request by tweet1992**
Adding KgPM and KgalPM

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/MassFlowTests.cs [hunk 1]
```diff
@@ -60,6 +60,8 @@ public class MassFlowTests : MassFlowTestsBase
 
         protected override double MegapoundsPerMinuteInOneGramPerSecond => 0.132277e-6;
 
+        protected override double KilogramsPerMinuteInOneGramPerSecond => 0.01666667e-3;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/MassFlowTests.cs]: [This source](http://www.kylesconverter.com/mass-flow/grams-per-second-to-kilograms-per-minute) says `0.060000000000000005`?

File: UnitsNet.Tests/CustomCode/VolumeFlowTests.cs [hunk 2]
```diff
@@ -96,6 +96,8 @@ public class VolumeFlowTests : VolumeFlowTestsBase
 
         protected override double OilBarrelsPerMinuteInOneCubicMeterPerSecond => 3.77388865E2;
 
+        protected override double KilousGallonsPerMinuteInOneCubicMeterPerSecond => 15850.323141489e-3;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: Correct number, but please use scientific notation: `1.5850323141489e1`. Verified with [source](https://www.unitjuggler.com/convert-flowrate-from-m3s-to-us-gpm.html).

File: UnitsNet.Tests/CustomCode/MassFlowTests.cs (hunk 1 above)
Comment (tweet1992) [reply UnitsNet.Tests/CustomCode/MassFlowTests.cs]: Good catch. I accidentally did the conversion the other way... Sorry!

File: UnitsNet.Tests/CustomCode/VolumeFlowTests.cs (hunk 2 above)
Comment (tweet1992) [reply UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: Fixed with latest commit

File: UnitsNet.Tests/CustomCode/VolumeFlowTests.cs [hunk 3]
```diff
@@ -96,7 +96,7 @@ public class VolumeFlowTests : VolumeFlowTestsBase
 
         protected override double OilBarrelsPerMinuteInOneCubicMeterPerSecond => 3.77388865E2;
 
-        protected override double KilousGallonsPerMinuteInOneCubicMeterPerSecond => 15850.323141489e-3;
+        protected override double KilousGallonsPerMinuteInOneCubicMeterPerSecond => 1.5850323141489e1;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: I think this value is correct, but the conversion function is perhaps wrong?

https://ci.appveyor.com/project/angularsen/unitsnet/builds/20278134
Values are not equal within relative tolerance: 0.0010 %
Expected: 15.850323141489
Actual: 15850323.141489
Diff: 99,999,900.0000 %
      Expected: True
      Actual:   False

Comment (angularsen) [reply UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: Also a side-question, is KiloUSGallonsPerMinute a widely used unit? I couldn't find any references to it when googling. Only found US gallons per minute, without kilo prefix.

Comment (tweet1992) [reply UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: I dont think it is widely used, but it is something that we are using in the oil and gas industry

Comment (tweet1992) [reply UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: FIxed a bug in the conversions :( Sorry for all the bugs! I'm glad you have a good continuous integration pipeline setup, lol.

Comment (angularsen) [reply UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: No problem! Yup, the builds sure help :-)

**Comment by angularsen** [comment]
Nuget 3.109 on the way out
https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F3.109.0

**Pull request merged**